api/v1alpha1/instaslice_types.go

```diff
@@ -36,6 +37,48 @@ const (
 	AllocationStatusCreated  AllocationStatusDaemonset  = "created"
 )
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// InstasliceOperator is the Schema for the InstasliceOperator API
+// +k8s:openapi-gen=true
+// +genclient
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+type InstasliceOperator struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	// spec holds user settable values for configuration
+	// +required
+	Spec InstasliceOperatorSpec `json:"spec"`
+	// status holds observed values from the cluster. They may not be overridden.
+	// +optional
+	Status InstasliceOperatorStatus `json:"status"`
+}
+
+// InstasliceOperatorSpec defines the desired state of InstasliceOperator
+type InstasliceOperatorSpec struct {
+	operatorv1.OperatorSpec `json:",inline"`
+
+	// EmulatedMode true configures the operator to not use the GPU backend
+	// +required
```

Comment:
nit: would it make sense for this to be optional?

I would expect almost everyone to not need to speicfy emulatedMode: false.